src/components/Comments/Comment.js

```diff
@@ -38,6 +46,7 @@ const Comment = ({
   onLongPress,
   updateVote
 }) => {
+  const refSheet = React.useRef();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `refSheet` is being defined but it's not clear what it's used for. It would be helpful to add a comment explaining its purpose.
```diff
+  // refSheet is used to control the BottomSheetMenu component
   const refSheet = React.useRef();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->